Sources/FingerprintJS/Harvesters/DataExchange/DeviceInfo.swift

```diff
@@ -38,6 +41,9 @@ public struct DeviceInfo: Equatable, Encodable {
     public let osRelease: String?
     public let kernelVersion: String?
 
+    /// Last device boot time in RFC3339 format.
+    public let bootTime: String?
```

Comment:
Given that `osInfoHarvester.bootTime` returns non-optional value:
```suggestion
    public let bootTime: String
```